src/styles/header.css

```diff
@@ -0,0 +1,70 @@
+/* Styles go here */
```

Comment:
The file is using CSS instead of SCSS. Rename the file to 'header.scss' and update the syntax to SCSS.